src/common/delivery/deploy/pcf/CloudFoundryApi.ts

```diff
@@ -201,6 +201,7 @@ export class CloudFoundryApi {
         }, {headers: _.assign({}, this.authHeader, this.jsonContentHeader)}),
             `create package ${appGuid}`);
         const formData = FormData();
+        formData.maxDataSize = Infinity;
```

Comment:
😱 
scary :-)

Reply:
Not that scary. We create that file that is being uploaded. If we are going to validate it (size or otherwise) we should probably do it on our own. We don't want the form data library to surprise us with its own ideas about file size limits.